styles/variables-silicon.less

```diff
@@ -130,14 +130,15 @@
 
 // Item
 // ---------------------------------------
-@agate-item-border-width:     3px;
-@agate-item-font-size:        27px;
-@agate-item-font-weight:      normal;
-@agate-item-height:           90px;
-@agate-item-height-large:     120px;
-@agate-item-label-font-size:  30px;
-@agate-item-side-outset:      30px;
-@agate-item-sub-text-size:    24px;
+@agate-item-border-width:        0;
+@agate-item-border-bottom-width: 3px;
```

Comment:
Because `box-sizing` is `border`, the Item width and height has reduced by removing the top, left, and right bottom width. Did you intent to do it? Could you check the GUI guide?

Reply:
I made a small change to `Item`'s height to correspond to GUI guide.